inc/profile.class.php

```diff
@@ -2839,6 +2900,26 @@ static function getHelpdeskItemtypes() {
    }
 
 
+   /**
+    * Get domains roecrds types
+    *
+    * @return array
+    */
+   public function getDomainRecordTypes() {
+      global $DB;
+
+      $iterator = $DB->request([
+         'SELECT' => '',
```

Comment:
Is this working ?

Reply:
Yes, we obtain a `SELECT *`, but this should be removed :)